ferris_cast_core/src/lib.rs

```diff
@@ -1,14 +1,76 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+use anyhow::{anyhow, Result};
+use chrono::prelude::DateTime;
+use chrono::{NaiveDate, ParseError, Utc};
+use rss::{Channel, Item};
+use std::convert::From;
+use std::fs::File;
+use std::io::BufReader;
+use std::path::PathBuf;
+use std::vec::Vec;
+
+pub static BAD_STRING: &str = "Unknown";
+
+pub struct Podcast {
+    pub title: String,
+    pub link: String,
+    pub image: rss::Image,      // TODO: If necessary add an image if needed
+    pub episodes: Vec<Episode>, // TODO: Implement episode type
+    pub last_build_date: String,
+}
+
+impl From<Channel> for Podcast {
+    fn from(channel: Channel) -> Self {
+        Podcast {
+            title: String::from(channel.title()),
+            link: String::from(channel.link()),
+            image: channel.image().unwrap().clone(),
```

Comment:
```suggestion
            image: channel.image().clone(),
```

Reply:
The below change to a string makes this change suggestion obsolete.